mongodb-replica-set-centos/azuredeploy.json

```diff
@@ -121,17 +137,10 @@
       "metadata": {
         "description": "The zabbix server IP which will monitor the mongodb nodes' mongodb status. Null means no zabbix server."
       }
-    },
-    "location": {
-      "type": "string",
-      "defaultValue": "[resourceGroup().location]",
-      "metadata": {
-        "description": "Location for all resources."
-      }
     }
   },
   "variables": {
-    "baseUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/mongodb-replica-set-centos/",
+    "baseUrl": "https://raw.githubusercontent.com/SychevIgor/azure-quickstart-templates/mongodb-replica-set-centos-fix/mongodb-replica-set-centos/",
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts

Reply:
@bmoore-msft it was addressed in a second commit a months ago